Code-4-Community/hands-across-the-sea-frontend
# Dl.visit reason dropdown

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/866744728/pulses/1290321186)



Allows dropdown selection of visit reason

## This PR



Created new reportData objects to handle custom other visit reason

## Screenshots

 

![image](https://user-images.githubusercontent.com/24307631/124335531-95214780-db68-11eb-8575-192d6de759c6.png)

![image](https://user-images.githubusercontent.com/24307631/124335558-abc79e80-db68-11eb-9c35-47d1b06243c0.png)


## Verification Steps

 

Npm run and selected a reason and verified it worked by looking at network (see screenshot)
